utils/json/src/sector.rs

```diff
@@ -65,20 +70,21 @@ pub mod json {
         .serialize(serializer)
     }
 
-    pub fn deserialize<'de, D>(deserializer: D) -> Result<PoStProof, D::Error>
+    pub fn deserialize<'de, D>(deserializer: D) -> Result<forest_shim::sector::PoStProof, D::Error>
     where
         D: Deserializer<'de>,
     {
         let m: JsonHelper = Deserialize::deserialize(deserializer)?;
-        Ok(PoStProof {
-            post_proof: RegisteredPoStProof::from(m.post_proof),
-            proof_bytes: BASE64_STANDARD
-                .decode(m.proof_bytes)
-                .map_err(de::Error::custom)?,
-        })
+        let reg_post_proof = <RegisteredPoStProof as Inner>::FVM::from(m.post_proof);
```

Comment:
I would prefer adding a `From` implementation for the shimmed `RegisteredPoStProof`.

Reply:
This too comes from upstream: https://github.com/filecoin-project/ref-fvm/blob/a1641680385d6b1b7c100abaf7bc67c0c11c0abf/shared/src/sector/registered_proof.rs#L305, that we may need to change as upstream changes, still do it?

Reply:
I'm not sure what you mean. Our `From` implementation would just defer to the FVM3 implementation.

Reply:
Added From impl

Reply:
Great. Let's use it.